yang/ietf-schedule.yang

```diff
@@ -91,20 +91,25 @@ module ietf-schedule {
             + ':[0-5][0-9]:[0-5][0-9]))|P([0-9]+)W';
     }
     description
-      "Duration of the time. The format can represent nominal
-       durations (weeks and days) and accurate durations (hours, 
-       minutes, and seconds). Note that this value type doesn't 
-       support the 'Y' and 'M' designators to specify durations in 
-       terms of years and months. Negative durations are typically 
-       used to schedule an alarm to trigger before an associated time.";
+      "Duration of the time as defined in Section 3.3.6 of RFC 5545. 
```

Comment:
```suggestion
      "Indicate a duration of the time. 
```

We need to avoid citing refs in the description as this is redundant with the reference statement.